server/src/main/java/com/google/sps/utils/AdRowProcessor.java

```diff
@@ -0,0 +1,116 @@
+/* 
+ * Description: Utility class that processes a CSV row and converts between row format and Ad format. 
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */ 
```

Comment:
Same as above. Javadoc goes on the class. For the other classes in the PR as well.